### PR TITLE
Update Broadcom plugin help for Zowe 1.28.1

### PR DESCRIPTION
**Pull request by t1m0thyj**
* Added the dbm-db2 plugin for Zowe CLI
* Updated help for endevor-bridge-for-git and ops plugins which seem like they had Zowe v2 docs in the v1 branch
* Updated help for caview plugin which moved some options from base to caview profile
* Other plugin updates should just be rebranding or updated copyright years

### CODE REVIEW

File: profiles/create/endevor-location.jsonc [hunk 1]
```diff
@@ -125,23 +125,13 @@
...
       "name": "maxrc",
-      "defaultValue": 8,
+      "defaultValue": 12,
       "group": "Options",
       "aliases": []
     },
-    {
-      "description": "Always override element signout, without having to specify the override signout option on each command",
-      "type": "boolean",
-      "name": "override-signout",
```
Comment (zFernand0) [inline profiles/create/endevor-location.jsonc:138]: Seems like Override Signout was added by some other version of the Endevor plugin.
And I assume it's being fixed here. 😋

File: profiles/update/caview.jsonc [hunk 2]
```diff
@@ -51,21 +51,23 @@
     {
       "type": "number",
       "name": "port",
-      "description": "Port of the target CA View REST API instance.",
+      "aliases": [
+        "P"
```
Comment (zFernand0) [inline profiles/update/caview.jsonc:55]: Alias added that was missed in a previous update, right ? 😋

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋 
A lot to go through, so I'm trusting the process for updating most of this 😉

**Review by KevinLoesch1 — APPROVED** [review]

File: commandGroups/endevor-bridge-for-git.jsonc [hunk 3]
```diff
@@ -1455,30 +1455,65 @@
       "positionals": []
     },
     {
-      "name": "endevor-connection",
+      "name": "endevor-credentials",
```
Comment (gejohnston) [inline commandGroups/endevor-bridge-for-git.jsonc:1458]: Does this seem like a V1 breaking change to anyone else?

Comment (gejohnston) [reply commandGroups/endevor-bridge-for-git.jsonc:1458]: I just re-read the PR description. This was probably backing something out.

Comment (t1m0thyj) [reply commandGroups/endevor-bridge-for-git.jsonc:1458]: Yes, there are a few changes in this PR that look breaking - especially for the endevor-bridge-for-git and ops plugins.

As far as I can tell, the zowe-v1-lts version of the endevor-bridge-for-git branch currently has and always has had an "endevor-credentials" group.

I believe that v2 web help was accidentally published for these plugins in the v1 branch at some point in the past. So while these changes appear breaking, they are really fixes to the web help.

**Pull request merged**